### ISSUE TITLE
[BITV] 9.4.1.2/4.5 - The "Open actions menu" menu has been implemented as an application menu with the role="menu".

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/photos#1687]
However, the implementation is not complete as the role="menuitem" for the individual menu items is missing. For more information, see, for example, https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions-active-descendant/. (1)

- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v4-n5

</details>

**Comment by Pytal** [comment]
Fixed on photos master
